test/e2e/storage/testsuites/volume_expand.go

```diff
@@ -259,24 +259,28 @@ func (v *volumeExpandTestSuite) defineTests(driver TestDriver, pattern testpatte
 // ExpandPVCSize expands PVC size
 func ExpandPVCSize(origPVC *v1.PersistentVolumeClaim, size resource.Quantity, c clientset.Interface) (*v1.PersistentVolumeClaim, error) {
 	pvcName := origPVC.Name
-	updatedPVC := origPVC.DeepCopy()
+	pvc := origPVC.DeepCopy()
 
 	waitErr := wait.PollImmediate(resizePollInterval, 30*time.Second, func() (bool, error) {
 		var err error
-		updatedPVC, err = c.CoreV1().PersistentVolumeClaims(origPVC.Namespace).Get(pvcName, metav1.GetOptions{})
+		pvc, err = c.CoreV1().PersistentVolumeClaims(origPVC.Namespace).Get(pvcName, metav1.GetOptions{})
 		if err != nil {
 			return false, fmt.Errorf("error fetching pvc %q for resizing with %v", pvcName, err)
 		}
 
-		updatedPVC.Spec.Resources.Requests[v1.ResourceStorage] = size
-		updatedPVC, err = c.CoreV1().PersistentVolumeClaims(origPVC.Namespace).Update(updatedPVC)
-		if err == nil {
-			return true, nil
+		pvc.Spec.Resources.Requests[v1.ResourceStorage] = size
+		updatedPVC, err := c.CoreV1().PersistentVolumeClaims(origPVC.Namespace).Update(pvc)
+		if err != nil {
+			// If an error happened, we don't return the Update()'d PVC
+			// because it might be invalid (e.g., with an empty name).
+			e2elog.Logf("Error updating pvc %s with %v", pvcName, err)
+			return false, nil
 		}
-		e2elog.Logf("Error updating pvc %s with %v", pvcName, err)
```

Comment:
This will be fixed once - https://github.com/kubernetes/kubernetes/pull/81107merges.

Reply:
Dropped commit